kube/manifests/thanos-store.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       containers:
       - name: thanos-store
         # Always use explicit image tags (release or master-<date>-sha) instead of ambigous `latest` or `master`.
-        image: improbable/thanos:v0.1.0
+        image: improbable/thanos:v0.2.1
```

Comment:
If we want to upgrade version here, we need to change ` "--gcs.bucket=<bucket-name>"` to `--objstore.config=<bucket.config-yaml>` as well (:

Reply:
Maybe adding a configmap exaplme also would be helpful?

Reply:
Sure. I don't have access to GCS. I am working on Openstack swift integration in lab environment. I will add that example once my configuration is completed.

Reply:
Or rather content copy (:

Reply:
anyway, let's add it later